summingbird-online/src/main/scala/com/twitter/summingbird/online/OnlineDefaultConstants.scala

```diff
@@ -41,6 +41,7 @@ private[summingbird] trait OnlineDefaultConstants {
   val DEFAULT_USE_ASYNC_CACHE = UseAsyncCache(false)
   val DEFAULT_ASYNC_POOL_SIZE = AsyncPoolSize(Runtime.getRuntime().availableProcessors())
   val DEFAULT_SOFT_MEMORY_FLUSH_PERCENT = SoftMemoryFlushPercent(80.0F)
+  val DEFAULT_COMPACT_VALUES = CompactValues(false)
```

Comment:
you can't add vals to a trait without breaking binary compatibility. This trips MIMA.

Can you move this to the style of:
```
case class CompactValues(toBoolean: Boolean)
object CompactValues {
  def default: CompactValues(false)
}
```

to avoid binary fragility?